packages/core/src/util/transactionSummaryInspector.ts

```diff
@@ -147,8 +151,34 @@ const toAssetInfoWithAmount = async (
  */
 export const transactionSummaryInspector: TransactionSummaryInspector =
   (args: TransactionSummaryInspectorArgs) => async (tx) => {
-    const { inputResolver, addresses, rewardAccounts, protocolParameters, assetProvider, dRepKeyHash } = args;
-    const resolvedInputs = await resolveInputs(tx.body.inputs, inputResolver);
+    const {
+      inputResolver,
```

Comment:
We should memoize inputResolver in lace as both inspectors are calling it with same inputs